routes/v1.js

```diff
@@ -2,15 +2,19 @@ import express from 'express';
 import UserController from '../controllers/user';
 import passport from '../auth/passport';
 import ProfileController from '../controllers/profile';
-import Auth from '../middleware/auth';
 import isUserVerified from '../middleware/verifyUser';
 
 import {
   validateSignup,
   validateLogin,
   validateProfileChange,
-  returnValidationErrors
+  validateEmail,
+  returnValidationErrors,
+  validatePassword
 } from '../middleware/validation';
+import PasswordReset from '../controllers/PasswordReset';
```

Comment:
This should be in the user controller

Reply:
I have moved the password reset methods to the user controller